fuse/archivemount/Portfile

```diff
@@ -2,28 +2,35 @@
 
 PortSystem              1.0
 PortGroup               fuse 1.0
+PortGroup               sourcehut 1.0
+PortGroup               legacysupport 1.1
+PortGroup               makefile 1.0
 
+# Epoch indicates new upstream, must now stay
+# See: https://trac.macports.org/ticket/70623
+sourcehut.setup         nabijaczleweli archivemount-ng 1
 name                    archivemount
-version                 0.9.1
-revision                1
+revision                0
+epoch                   1
```

Comment:
The new version `1` is larger than the previous version `0.9.1` so increasing the epoch shouldn't be necessary.

Reply:
I think I realized this at some point but entirely forgot. Will be fixed in latest commit.